src/hooks/useConfigure.ts

```diff
@@ -53,16 +50,16 @@ export function useConfigure({
    * Set theme mode
    */
   useEffect(() => {
-    ThemeCtrl.setThemeMode(themeMode || colorScheme);
-  }, [themeMode, colorScheme]);
+    ThemeCtrl.setThemeMode(config.themeMode || colorScheme);
+  }, [config.themeMode, colorScheme]);
 
   /**
    * Set config
    */
   useEffect(() => {
-    ConfigCtrl.setProjectId(projectId);
-    ConfigCtrl.setMetadata(providerMetadata);
-  }, [projectId, providerMetadata]);
+    ConfigCtrl.setConfig(config);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
is this disable still needed? since setConfig is outside depp, maybe eslint will not scream. Just checking.

Reply:
`setConfig` is outside depp, but `config` is a param received in the hook -> `useConfigure(config: Props)`